README.md

```diff
@@ -106,11 +106,9 @@ mabl applicationId: 'APP-ID-a', continueOnMablError: true, continueOnPlanFailure
 
 ``` syntaxhighlighter-pre
 node {
-   stage('mabl') {
-       steps {
-           step([$class: 'MablStepBuilder', restApiKeyId: 'REST-API-KEY-ID', environmentId: 'ENV-ID-e', applicationId: 'APP-ID-a'])
-       }
-   }
+  stage('mabl') {
```

Comment:
Are we sure this is the correct format? Looking at the [Jenkins Pipeline docs](https://www.jenkins.io/doc/book/pipeline/), all the examples I see have `stage > stages > steps > step`. Are we sure it is OK to remove the `steps` block?

```
stages {
        stage('Build') { 
            steps { 
                sh 'make' 
            }
        }
    }
```

From the Jenkins [docs](https://www.jenkins.io/doc/book/pipeline/syntax/#stage)
> The stage directive goes in the stages section and should contain a steps section

Reply:
@twistedpair Thank you for your comment. My customer and I tried to use your sample but we faced "java.lang.NoSuchMethodError: No such DSL method 'steps' found among steps". So DSL issue or the example is wrong.

Please check this link: https://stackoverflow.com/questions/42113655/no-such-dsl-method-stages

This example does not use "steps" for scripted pipeline too.

Reply:
One more information.

You can see this page for scripted pipeline(old format): https://www.jenkins.io/doc/book/pipeline/syntax/#scripted-steps

We can see this part.

>As discussed at the start of this chapter, the most fundamental part of a Pipeline is the "step". Fundamentally, steps tell Jenkins what to do and serve as the basic building block for both Declarative and Scripted Pipeline syntax.
>Scripted Pipeline does not introduce any steps which are specific to its syntax; Pipeline Steps reference contains a comprehensive list of steps provided by Pipeline and plugins.